tf/main.tf

```diff
@@ -55,7 +55,7 @@ module "lambda_layer_local" {
   description         = "AWS Lambda Extension Layer for Elastic APM - smoke testing"
   compatible_runtimes = ["nodejs16.x"]
 
-  source_path = "../bin/"
+  source_path = "../dist/apm-lambda-extension_linux_amd64_v1/"
```

Comment:
will this particular name change in the future? If so, what do you think adding a comment regarding where this name is coming from?

Reply:
There is actually a comment about this in the GoRelease documentation.. will find and add it here.

But from my understanding, this will only change if changing the project name OR potentially when upgrading the goreleaser version.

Reply:
https://goreleaser.com/customization/build/#a-note-about-folder-names-inside-dist

> By default, GoReleaser will create your binaries inside dist/${BuildID}_${BuildTarget}, which is an unique directory per build target in the matrix.
>
> Those names have no guarantees of remaining the same from one version to another. If you really need to access them from outside GoReleaser, you should be able to consistently get the path of a binary by parsing dist/artifacts.json.

Reply:
https://github.com/elastic/apm-aws-lambda/pull/370/commits/a1efdaf3034d5b5c9b759794c1123955a3754237